angular/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## v6.0.0 (Not published)
 
+> Migrating some 5.0.0 to 6.0.0 may require higher specificity when applying custom styles to Angular or PX Blue components.
```

Comment:
```suggestion
> Migrating to version 6.0.0 may require you to provide higher specificity if you were applying custom styles to Angular or PX Blue components.
```

Reply:
I would move this into the changed section so we keep our consistent changelog format